packages/venia-concept/src/RootComponents/Category/category.js

```diff
@@ -1,51 +1,83 @@
 import { Component, createElement } from 'react';
-import PropTypes from 'prop-types';
-
+import { string, number, shape } from 'prop-types';
+import { Query } from 'react-apollo';
+import gql from 'graphql-tag';
 import classify from 'src/classify';
 import Gallery from 'src/components/Gallery';
 import Page from 'src/components/Page';
-import mockData from './mockData';
 import defaultClasses from './category.css';
 
-const emptyData = [];
+const categoryQuery = gql`
+    query category($id: Int!) {
+        category(id: $id) {
+            description
+            name
+            product_count
+            products {
+                items {
+                    id
+                    name
+                    small_image
+                    price {
+                        regularPrice {
+                            amount {
+                                value
+                                currency
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+`;
 
 class Category extends Component {
     static propTypes = {
-        classes: PropTypes.shape({
-            gallery: PropTypes.string,
-            root: PropTypes.string,
-            title: PropTypes.string
+        id: number,
```

Comment:
Likewise, can `id` be a `string`?

Reply:
See https://github.com/magento-research/pwa-studio/pull/52#discussion_r197236280